lib/oidcstrategy.js

```diff
@@ -187,10 +272,16 @@ Strategy.prototype.authenticate = function authenticateStrategy(req, options) {
 
   async.waterfall(
     [
-      // compute metadata url
+      /* 
+       * Step 1. compute metadata url 
+       */
       (next) => {
         // B2C interception
-        let metadata = self._options.identityMetadata;
+        let metadataUrl = self._options.identityMetadata;
+
+        // use 'ordinary' as the default cachekey, in the case of B2C, we will
+        // change the value to policy later
+        let cachekey = 'ordinary';
```

Comment:
Can we make the values 'ordinary' and 'policy' constants defined earlier in the file so that it is clearer what all of the options are for the cachekey?